docs/usage.md

```diff
@@ -272,6 +272,10 @@ person.address.populator:
 Be aware - that both properties have the same name should not lead you think they have the same type.
 There is really an object conversion behind done by `address.converter`.
 
+If you set the `sourcePropertyName` as empty string the complete `source` object will be used for population.
+
+Especially in addition wih the `ConvertingPopulator` this is sometimes necessary.
```

Comment:
```suggestion
Especially in connection with the `ConvertingPopulator` this is sometimes necessary.
```

But... do we really need this sentence? This whole paragraph is about the `ConvertingPopulator`.

Maybe we should rather say instead that this is only possible with the `PropertyMappingPopulator`.